sycl/unittests/pi/BackendString.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <detail/plugin.hpp>
+
+using namespace cl::sycl;
```

Comment:
Hm... I would remove this from this header file and from sycl/unittests/pi/cuda/test_get_plugin.hpp.

Reply:
Agreed! Wee copy paste mistake there. Well spotted.

Reply:
Done